install-and-configure/advanced-configuration/network-costs-configuration.md

```diff
@@ -37,29 +37,26 @@ networkCosts:
 
 You can view a list of common config options [here](https://github.com/kubecost/cost-analyzer-helm-chart/blob/700cfa306c8e78bc9a1039b584769b9a0e0757d0/cost-analyzer/values.yaml#L573).
 
-### Prometheus:
+### Prometheus
 
 * If using Kubecost-bundled Prometheus instance, the scrape is automatically configured.
 * If you are integrating with an existing Prometheus, you can set `networkCosts.prometheusScrape=true` and the network costs service should be auto-discovered.
 * Alternatively, a serviceMonitor is also [available](https://github.com/kubecost/cost-analyzer-helm-chart/blob/700cfa306c8e78bc9a1039b584769b9a0e0757d0/cost-analyzer/values.yaml#L716).
 
-### Log Level:
+### Log Level
 
-*   You can adjust log level using the `extraArgs` config:
+* You can adjust log level using the `extraArgs` config:
 
     ```yaml
     networkCosts:
       enabled: true
       extraArgs:
         - "-v=0"
     ```
```

Comment:
```suggestion
```yaml
networkCosts:
  enabled: true
  extraArgs:
    - "-v=0"
    ```
```

Reply:
I believe this code block requires the extra indentation because it's within a bulleted list. Checking the rendered [Gitbook revision here](https://gitbook.kubecost.com/~/revisions/GLU415OZnhw9YHh6B334/install-and-configure/advanced-configuration/network-costs-configuration#log-level), it looks like changing this code block may have broken things.

Reply:
@chipzoller I'm going to re-indent this code block. That ok?

Reply:
![Screenshot 2023-10-31 at 11 08 52 AM](https://github.com/kubecost/docs/assets/11251627/d9b73b58-bce4-432b-b083-2dba675a239b)

Reply:
I am sorry about that. Must be one of the peculiarities of GitBook.